packages/lib/shim-init-node.js

```diff
@@ -97,6 +98,10 @@ function shimInit(options = null) {
 		nodeSqlite: null,
 		...options,
 	};
+	// Work around issues with ipv6 resolution
+	// (possibly incorrect URL serialization see https://github.com/mswjs/msw/issues/1388#issuecomment-1241180921).
+	// See also https://github.com/node-fetch/node-fetch/issues/1624#issuecomment-1407717012
+	dns.setDefaultResultOrder('ipv4first');
```

Comment:
This is currently done in `shim-init-node` so that it applies to the CLI app as well (there may be a better place to do this).

Reply:
Note: This should be moved to BaseApplication.ts